bin/wazo-service

```diff
@@ -152,7 +152,7 @@ start_services() {
     local services=$1
     for service in $services; do
         if [ "$service" = postgresql ]; then
-            service=postgresql@13-main
+            service=postgresql@15-main
```

Comment:
What do you think about adding a variable e.g. `$POSTGRESQL_SERVICE`?

Could be reused by the python script too.

Reply:
No objection
For bigger change (ex variable injection)
I would create another ticket, to keep the bookworm change smaller as possible (avoid issue, easier to rebase, etc..)